server/app.js

```diff
@@ -42,6 +43,7 @@ if (process.env.NODE_ENV === 'development') {
   // colorful output for dev environment
   app.use(morgan('dev'));
 }
+app.use(cors());
```

Comment:
I would add cors only to the routes where you need it, with this every route is accessible from outside, you can add it as middleware on specific routes if that is needed for sockets